types/wallet.d.ts

```diff
@@ -29,10 +36,10 @@ export declare class BaseWallet implements Wallet {
     importExtendedPrivateKey(xpriv: string): Promise<void>;
     importWif(wif: string): Promise<void>;
     update(): Promise<void>;
-    broadcast(tx: tapyrus.Transaction): Promise<string>;
+    broadcast(tx: tapyrus.Transaction, options?: BroadcastOptions): Promise<string>;
```

Comment:
If it is a communication parameter that is not directly related to the operation of the wallet, it is better to be able to get it internally as a configuration value, not as an argument.

Reply:
This is intended that backend application needs additional parameter such as "Receipt ID" or "Gift Code" that differ between each transaction. So i think it is better to add parameters as an argument than configuration